api/app/controllers/spree/api/v1/product_properties_controller.rb

```diff
@@ -30,23 +30,19 @@ def create
         end
 
         def update
-          if @product_property
-            authorize! :update, @product_property
-            @product_property.update_attributes(product_property_params)
-            respond_with(@product_property, status: 200, default_template: :show)
-          else
-            invalid_resource!(@product_property)
```

Comment:
@yu-liang-kono I think we should keep the `invalid_resource!` call in this action, this is needed in case product_property is invalid. :)